robotiq_driver/src/gripper_interface_test.cpp

```diff
@@ -35,58 +35,61 @@ constexpr auto kComPort = "/dev/ttyUSB0";
+        
+        std::cout << "Gripper successfully activated.\n";
 
-    std::cout << "Closing gripper...\n";
-    gripper.setGripperPosition(0xFF);
-    while (gripper.gripperIsMoving()) {
-        std::this_thread::sleep_for(std::chrono::milliseconds(500));
-    }
+        std::cout << "Closing gripper...\n";
+        gripper.setGripperPosition(0xFF);
+        while (gripper.gripperIsMoving()) {
+            std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        }
 
-    std::cout << "Opening gripper...\n";
-    gripper.setGripperPosition(0x00);
-    while (gripper.gripperIsMoving()) {
-        std::this_thread::sleep_for(std::chrono::milliseconds(500));
-    }
+        std::cout << "Opening gripper...\n";
+        gripper.setGripperPosition(0x00);
+        while (gripper.gripperIsMoving()) {
+            std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        }
 
-    std::cout << "Half closing gripper...\n";
-    gripper.setGripperPosition(0x80);
-    while (gripper.gripperIsMoving()) {
-        std::this_thread::sleep_for(std::chrono::milliseconds(500));
-    }
+        std::cout << "Half closing gripper...\n";
+        gripper.setGripperPosition(0x80);
+        while (gripper.gripperIsMoving()) {
+            std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        }
 
-    std::cout << "Opening gripper...\n";
-    gripper.setGripperPosition(0x00);
-    while (gripper.gripperIsMoving()) {
-        std::this_thread::sleep_for(std::chrono::milliseconds(500));
-    }
+        std::cout << "Opening gripper...\n";
+        gripper.setGripperPosition(0x00);
+        while (gripper.gripperIsMoving()) {
+            std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        }
 
-    std::cout << "Decreasing gripper speed...\n";
-    gripper.setSpeed(0x0F);
+        std::cout << "Decreasing gripper speed...\n";
+        gripper.setSpeed(0x0F);
 
-    std::cout << "Closing gripper...\n";
-    gripper.setGripperPosition(0xFF);
-    while (gripper.gripperIsMoving()) {
-        std::this_thread::sleep_for(std::chrono::milliseconds(500));
-    }
+        std::cout << "Closing gripper...\n";
+        gripper.setGripperPosition(0xFF);
+        while (gripper.gripperIsMoving()) {
+            std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        }
 
-    std::cout << "Increasing gripper speed...\n";
-    gripper.setSpeed(0xFF);
+        std::cout << "Increasing gripper speed...\n";
+        gripper.setSpeed(0xFF);
 
-    std::cout << "Opening gripper...\n";
-    gripper.setGripperPosition(0x00);
-    while (gripper.gripperIsMoving()) {
-        std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        std::cout << "Opening gripper...\n";
+        gripper.setGripperPosition(0x00);
+        while (gripper.gripperIsMoving()) {
+            std::this_thread::sleep_for(std::chrono::milliseconds(500));
+        }
+    } catch(const std::runtime_error &e){
```

Comment:
same here, apply to whole PR

```suggestion
    } catch (const std::runtime_error &e) {
```